components/avatar.json

```diff
@@ -1,6 +1,7 @@
 {
   "avatar": {
-    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4594%3A8792",
+    "comment":"applied to avatars and all its elements",
+    "figma":"https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=4389%3A1647",
```

Comment:
These are different links, but appear to go to the same place. Figma is confusing

Reply:
the position of the view is slightly different; that is why the link is different. They both take you to the correct component, so Ill mark it as resolved